src/pages/background/userActions.js

```diff
@@ -155,7 +155,7 @@ export function logOut() {
 }
 
 export function loggedOutOfPocket() {
-  removeSettings(['access_token'])
+  chrome.storage.local.clear()
```

Comment:
Easy-peasy

Reply:
Is `removeSettings` being used anywhere else?

Reply:
Nope! good call. I deleted it from `interface`